src/coordinator/index.js

```diff
@@ -58,6 +67,69 @@ class Coordinator {
     this.edgeNodes = new EdgeNodes(sdk, request, baseUrl);
   }
 
+  /**
+   * Creates an application to a user's favorited applications
+   *
+   * API Endpoint: '/applications/:applicationId/favorites'
+   * Method: POST
+   *
+   * @param {number} applicationId The ID of the application
+   *
+   * @returns {Promise}
+   * @fulfill {ContxtUserFavoriteApplication} Information about the contxt application favorite
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.coordinator
+   *   .createFavoriteApplication(25)
+   *   .then((favoriteApplication) => console.log(favoriteApplication))
+   *   .catch((err) => console.log(err));
+   */
+  createFavoriteApplication(applicationId) {
+    if (!applicationId) {
+      return Promise.reject(
+        new Error(
+          'An application ID is required for creating a favorite application'
+        )
+      );
+    }
+
+    return this._request
+      .post(`${this._baseUrl}/applications/${applicationId}/favorites`)
+      .then((favoriteApplication) => toCamelCase(favoriteApplication));
+  }
+
+  /**
+   * Deletes an application from a user's favorited applications
```

Comment:
What about `Removes an application from a user's list of favorited applications`?

Reply:
👍